### PR TITLE
Meeting emails

### PR DESCRIPTION
**Pull request by mlgiardina**
fixed meetings controller to now assign a room id

### CODE REVIEW

File: app/views/meeting_mailer/meeting_cancelled.txt.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@ Hello <%=@employee.name%>,
 This email is to inform you that the following meeting has been cancelled:
 
     Title: <%= @meeting.title %>
-    Location: <%= @meeting.location %>
+    Location: <%= @meeting.room.location %>
```
Comment (JMD1986) [inline app/views/meeting_mailer/meeting_cancelled.txt.erb:6]: good catch

**Comment by tclewis91** [comment]
:+1:

**Comment by JMD1986** [comment]
mui bueno :+1:

**Comment by BenHand** [comment]
:+1: :8ball:

**Pull request merged**